modules/drivers/redshift/deps.edn

```diff
@@ -5,4 +5,4 @@
  {"redshift" {:url "https://s3.amazonaws.com/redshift-maven-repository/release"}}
 
  :deps
- {com.amazon.redshift/redshift-jdbc42 {:mvn/version "2.1.0.3"}}}
+ {com.amazon.redshift/redshift-jdbc42 {:mvn/version "2.1.0.5"}}}
```

Comment:
Could we upgrade to latest 2.1.0.9 while we're at it? https://github.com/aws/amazon-redshift-jdbc-driver/blob/master/CHANGELOG.md